src/actions/_rsaa.js

```diff
@@ -1,8 +1,61 @@
+import _ from 'lodash';
+
+/**
+ * Check if headers object supports entities() method.
+ * @param {Headers} headers
+ */
+function isHeadersEntriesSupported(headers) {
+  return (_.isFunction(headers.entries));
+}
+
+/**
+ * Return existing or create new Headers object instance
+ * @param {Headers|Object} headers
+ */
+function getHeadersInstance(headers) {
+  if (headers instanceof Headers) {
+    return headers;
+  }
+  return new Headers({ ...headers });
+}
+
+/**
+ * Extract response headers into a plan object
+ * @param {Object} response
+ */
+export function extractResponseHeaders(response) {
+  if (!response || !response.headers) {
+    return {};
+  }
+
+  const headersInstance = getHeadersInstance(response.headers);
+  const headers = {};
+  let currentHeader = null;
```

Comment:
Code is easier to understand, doesn't change value etc. This is critical part of Shoutem platform and should be robust as much as possible. 
https://softwareengineering.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6